frontend/src/pages/detail/PODetail.tsx

```diff
@@ -0,0 +1,251 @@
+import React, { FormEvent } from "react";
+import { Button } from "primereact/button";
+import { InputText } from "primereact/inputtext";
+import { ToggleButton } from "primereact/togglebutton";
+import { Calendar, CalendarProps } from "primereact/calendar";
+import { Dropdown, DropdownProps } from "primereact/dropdown";
+import { DataTable } from "primereact/datatable";
+import { TableColumn } from "../../components/Table";
+import { Column, ColumnEditorOptions } from "primereact/column";
+import { InputNumber } from "primereact/inputnumber";
+import ConfirmButton from "../../components/ConfirmButton";
+
+interface modifyPOState {
+  date: any;
+  vendor: string;
+  checked: boolean;
+  confirmationPopup: boolean;
+}
+
+interface Vendors {
+  name: string;
+  code: string;
+}
+
+const dataVendors: Vendors[] = [
+  { name: "New York", code: "NY" },
+  { name: "Rome", code: "RM" },
+  { name: "London", code: "LDN" },
+  { name: "Istanbul", code: "IST" },
+  { name: "Paris", code: "PRS" },
+];
+
+interface POlineRow {
+  books: string;
+  quantity: number;
+  unitRetailPrice: number;
+}
+
+const dataPO: POlineRow[] = [
```

Comment:
I'd maybe change this to "POPurchaseRow" or something like that, and "SRSaleRow similarly"

Reply:
Also these constants should be in all caps